reflections.md

```diff
@@ -1,3 +1,45 @@
-# Reflections on Collaborative Coding
-
-Write your reflections here using [GFM (Github Flavored Markdown)](https://github.github.com/gfm/)
+# Reflections on the Earthquake Lab
+
+I was paired with Jesse and Gerry, both of which are experienced programmers than me. I did my best to keep up with the both of them during the 
+
+process. It would have taken me a whole lot longer to start pulling from APIs but observing them taught me a whole lot about the process. 
+
+The way they set up the code was really well thought out. Gerry took more of a well thought out approach while Jesse just ripped into to 
+
+problems. At some point i was having difficulty contributing much at all because before my brain could arrive at a conclusion they had solved 
+
+the issue. I did catch a pretty big error in our code early on but they were so deep into another part of it that I wasn't really heard when I 
+
+spoke up. I assumed that they knew about the issue and we're just testing out some other things before wrapping up that part of the function 
+
+they were building. Because I'm new to web development in general I lack confidence to speak up in the presence of experienced programmers. I 
+
+learned that I should be more vocal about an issue even if I assume that they picked up on it already, because both Jesse and Gerry's minds 

```

Comment:
YES YES YES! And both of your partners mentioned this as well. Just because you think you're less experienced doesn't mean you can't contribute. Learning how to gain the confidence needed to speak up is hard, but once you can get past the fear of being 'wrong' or 'stupid',  pair programming will become SO much more successful for you! Awesome realization!